subprojects/extTest/extTest.gradle

```diff
@@ -23,3 +25,7 @@ configurations.all {
     //let's make those tests not use hamcrest
     //exclude group: 'org.hamcrest', module: 'hamcrest-core'
 }
+
+jacoco {
+    toolVersion = libraries.jacocoVersion
```

Comment:
Do we really need it to be defined in every module separately? Maybe you could use the list of subprojects (I proposed above) to do it in a more generic way?

Reply:
Yes, because only this task will generate the correct `*.exec` files in `build/` which then have to be picked up by the root project. If you have a better solution, feel free to push to this PR!

Reply:
I will try to tune that tomorrow. Stay tuned.

Reply:
Let's avoid code duplication. I'm sure @szpak will figure this out.

Reply:
Sure, if not I will ask the Gradle 2.x core developer for help :).

WiP.

Reply:
It was more tricky and took much more time that I expected... I did it in 3 somehow different ways and in the end I left the simplest version (with one file gathering all coverage data). Feel free to comment and to enhance.